lib/Runtime/Library/ThrowErrorObject.cpp

```diff
@@ -18,7 +18,7 @@ namespace Js
         ThrowErrorObject* throwErrorObject = ThrowErrorObject::FromVar(function);
 
         bool useExceptionWrapper =
-            scriptContext->IsInDebugMode() &&
+            scriptContext->IsScriptContextInDebugMode() &&
```

Comment:
Add a comment that it is intentional, the library code is also using the exception wrapper.